### DISCUSSION TITLE
Just need a simple click to download to use - just a Linux end user on Synaptic package manager. Is there a file to download or just something in development? Not looking to get complicated or a cut a pasted for terminal? I was just searching for Google Maps for Linux but found this.

### DISCUSSION THREAD

**Discussion by JoeCreed** [OvertureMaps/data#331] [Discussion]
Just need a simple click to download to use - just a Linux end user on Synaptic package manager. Is there a file to download or just something in development? Not looking to get complicated or a cut a pasted for terminal? I was just searching for Google Maps for Linux but found this.

**Comment by jwass** [comment]
Hi @JoeCreed - OvertureMaps isn't a mapping application like Google Maps. We provide underlying geospatial data for maps service providers and other developers to build their own applications. Unfortunately I don't think we provide what you're looking for.

**Reply by JoeCreed** [reply]
Thanks for the info, sorry for the misunderstanding. If you know of any Ubuntu based mapping application/program that uses your data for the general non Google community please forward my contact information if at all possible.   Again sorry for the misunderstanding and thanks for the speedy reply
